README.md

```diff
@@ -26,17 +26,32 @@ This project provides an API for storing and retrieving learning resources that
 
 Sometimes these installs can be tricky.  If you get stuck ask for help in the Slack [#oc-python-projects](https://operation-code.slack.com/messages/C7NJLCCMB) channel!
 
+1. If you are not a member of Operation Code, please sign up at https://operationcode.org/join
+    - Provide your email, name, zip code, and a password of your choosing.
+    - You will need this email and password later when creating your API key.
+    - Look for a Slack invite email and join the Slack organization, and then the [#oc-python-projects](https://operation-code.slack.com/messages/C7NJLCCMB) channel.
+
 1. Install [Git](https://git-scm.com/downloads).
-- Choose your OS from the website and follow the prompts.  This installs Git and the Bash Terminal on your machine.
-- Extra: [Git Documentation](https://git-scm.com/doc) for more information on Git.
-2. Fork & Clone
-- [Fork a repository](https://help.github.com/articles/fork-a-repo/)
-- Create a local clone of your fork (Step 2 in the document above)
-3. Install Docker [Mac and Windows](https://www.docker.com/products/docker-desktop) or [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/) and ensure it is running
-- Linux: install [docker compose](https://docs.docker.com/compose/install/#install-compose) as well.
-4. [Install Make](http://gnuwin32.sourceforge.net/packages/make.htm) if you're on Windows. OSX already has it installed. Linux will tell you how to install it (i.e., `sudo apt-get install make`)
-5. Run `make setup`
-6. Run `make all` and then navigate to http://localhost:5000/api/v1/resources
+    - Choose your OS from the website and follow the prompts.  This installs Git and the Bash Terminal on your machine.
+    - Windows users: use the Git Bash Terminal for any of
+      the commands in the remainder of this README.
+      [The Bash Primer](http://www.compciv.org/bash-guide/)
+    - Extra: [Git Documentation](https://git-scm.com/doc) for more information on Git.
+
+1. Fork & Clone
+    - [Fork a repository](https://help.github.com/articles/fork-a-repo/)
+    - Create a local clone of your fork (Step 2 in the document above)
```

Comment:
I know this was here before you got to it, but while we're in here do you think we could either remove or clarify the `Step 2` comment? I think there was a point in time where this was stated in a further step down, but now that it's a part of step 2 it seems really unnecessary to even mention which step is referring to the fork.

Reply:
Also, it looks like it's step 3 now after all

Reply:
Removed it... agreed it's unnecessary.